pkg/authenticator/authenticator.go

```diff
@@ -203,6 +203,8 @@ func (auth *Authenticator) IsCertExpired() bool {
 // Authenticate sends Conjur an authenticate request and returns
 // the response data. Also manages state of certificates.
 func (auth *Authenticator) Authenticate() ([]byte, error) {
+	log.Debug(log.CAKC040, auth.Config.Username)
```

Comment:
Maybe add these logs on top of the info logs?
These info logs are the only logs written when debug logs are off, which is the default.

Reply:
i don't understand. would you log these both in INFO and in DEBUG?

Reply:
I think you can add any debug logs you think is necessary, just don't remove the only info logs.

Reply:
ok switched these back to INFO